File: pset6/mario.py
import cs50
# get valid height from user
print("Height: ", end= "")
while True:
    h = cs50.get_int()
    if h > 0 or h < 23:
        break
# print pattern for height
for i in range(h):
    for s in range((h-1) - i):
        print(" ", end= "")
    for p in range(i + 2):
        print("#", end="")
    print()

File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int height;
int space;
int count;
int hash;
int main(void)
{
    do
    {
        printf("Height: ");
        height = get_int();
    }
    while ((height < 0) || (height > 23));
    for (count = 0; count < height; count++)
    {
        for (space = ((height - 1) - count); space > 0; space--)
        {
            printf(" ");
        }
        for (hash = 0; hash < (count + 2); hash++)
        {
            printf("#");
        }
        printf("\n");
    }
}